---
content_type: page
description: Assignment to write a paper and present it in class.
draft: false
learning_resource_types:
- Assignments
ocw_type: CourseSection
parent_title: Assignments
parent_type: CourseSection
parent_uid: 5a2e497f-e49c-5201-6e67-a52f578758e7
title: 'Module 1 Assignments: Journal Club'
uid: 96d78713-6ddb-e72b-2a7f-724d9ac77ed6
video_files:
  video_thumbnail_file: null
video_metadata:
  youtube_id: null
---
## Logistics of Paper Sign-Up and Presentation

Once you have decided on a paper for your presentation, please "reserve" it by signing up on the class list. If you would like to discuss a paper not on the list below, please email it to the instructor’s email list with a brief description.

As you prepare your talk be sure to follow the class {{% resource_link "c054e983-55c4-7571-51ab-8f1efea6beb0" "guidelines for oral presentations" %}} in this class.

Day 6 presentations will begin after lab work is finished. The entire class session on Day 8 will be devoted to presentations.

## Paper Options

The list of papers below is provided as a guideline for the types of papers that might be relevant for your presentation. You are not limited to the primary research articles on this list. The list is provided simply to give you an idea of the kinds of subjects that could make suitable presentations for the class. Search {{% resource_link "37d61b47-0d97-410f-a701-67d9190a6c61" "PubMed" %}} yourself to find articles of interest to you.

Note: Some of these articles are open access, with full text online as linked in the following citations. Other papers, whose full text is available only to subscribers or subscribing institutions, are linked below to the abstracts.

### Methodology/Technology

1. Brudno, Y., et al. "{{% resource_link "5a8bd333-8c25-461e-953e-d51db3a04eac" "An *in vitro* Translation, Selection and Amplification System for Peptide Nucleic Acids" %}}." *Nat Chem Biol* 6, no. 2 (2010): 148-55
2. Muranaka, N., et al. "{{% resource_link "3a77faaa-e0d7-494d-82b0-57057e220359" "Mechanism-guided Library Design and Dual Genetic Selection of Synthetic OFF Riboswitches" %}}." *Chembiochem* 10, no. 14 (2009): 2375-81
3. König, J., et al. "{{% resource_link "998c1903-c2a4-4bb3-a6fb-df333dbf264d" "Combining SELEX and the Yeast Three-hybrid System for *in vivo* Selection and Classification of RNA Aptamers" %}}." *RNA* 13, no. 4 (2007): 614-22. \[Open Access\]
4. Lou, X., et al. "{{% resource_link "2ee703ca-fe6d-49b5-b6be-8c075fdc5195" "Micromagnetic Selection of Aptamersin Microfluidic Channels" %}}." *PNAS* (2009): 1-6. \[Open Access\]
5. Berezovski, M., et al. "{{% resource_link "a4edfef7-cf7f-4abf-bdf3-1e8625bb05e8" "Nonequilibrium Capillary Electrophoresis of Equilibrium Mixtures: A Universal Tool for Development of Aptamers" %}}." *J Am Chem Soc* 127, no. 9 (2005): 3165-71
6. Vuyisich, M., and P. A. Beal. "{{% resource_link "35409708-39c1-47d3-ab57-3ab0908cc206" "Controlling Protein Activity with Ligand-regulated RNA Aptamers" %}}." *Chem Biol* 9, no. 8 (2002): 907-13
7. Cox, J. C, and A. D. Ellington. "{{% resource_link "63293c97-2eda-47de-bbc0-443ea112c585" "Automated Selection of Anti-protein Aptamers" %}}." *Bioorg Med Chem* 9, no. 10 (2001): 2525-31.
8. Mi, J., et al. "{{% resource_link "785f1dd3-e8b3-40bd-98ce-08e0a4050d4e" "*In vivo* Selection of Tumor-targeting RNA Motifs" %}}." *Nat Chem Biol* 6, no. 1 (2010): 22-4 \[Open Access\]

### Applications

1. Wu, Y., et al. "{{% resource_link "01c07532-5181-4d44-8e0b-9d6744522d17" "DNA Aptamer–micelle as an Efficient Detection/delivery Vehicle Toward Cancer Cells" %}}." *PNAS* 107, no. 1 (2010): 5-10. \[Open Access\]
2. Mayer, G., et al. "{{% resource_link "b5d040cd-b1d0-4117-94b0-823521eabd7c" "An RNA Molecule that Specifically Inhibits G-protein-coupled Receptor Kinase 2 *in vitro*" %}}." *RNA* 14, no. 3 (2008): 524-34. \[Open Access\]
3. Homann, M., et al. "{{% resource_link "6473c97c-c2dd-4284-99ab-bb463fc943ac" "Serum-stable RNA Aptamers to an Invariant Surface Domain of Live African Trypanosomes" %}}." *Comb Chem High Throughput Screen* 9, no. 7 (2006): 491-9
4. An, C. I., et al. "{{% resource_link "21972e39-1dad-42d2-85eb-0b4e1b98f441" "Artificial Control of Gene Expression in Mammalian Cells by Modulating RNA Interference through Aptamer-small Molecule Interaction" %}}." *RNA* 12, no. 5 (2006): 710-6. \[Open Access\]
5. Hicke, B. J., et al. "{{% resource_link "4688f412-fb46-42ac-a404-6ed1b69a50de" "Tumor Targeting by an Aptamer" %}}." *J Nucl Med* 47, no. 4 (2006): 668-78 \[Open Access\]
6. Cerchia, et al. "{{% resource_link "b9cd43bb-82aa-40eb-b6eb-638af9e2efff" "Neutralizing Aptamers from Whole-cell SELEX Inhibit the RET Receptor Tyrosine Kinase" %}}." *PLoS Biol* 3, no. 4 (2005): e123 \[Open Access\]
7. Collett, J. R., et al. "{{% resource_link "b0c713c0-8f4d-4de2-b674-4ff8ecc54e67" "Functional RNA Microarrays for High-throughput Screening of Aantiprotein Aptamers" %}}." *Anal Biochem* 338, no. 1 (2005): 113-23.
8. Jian, Y., et al. "{{% resource_link "2e6db5f7-277c-4a96-a0ec-30b4a65401e5" "RNA Aptamers Interfering with Nucleophosmin Oligomerization Induce Apoptosis of Cancer Cells" %}}." *Oncogene* 28, no. 1 (2009): 4201-4211

### Aptamers in Nature

1. Mandal, M., et al. "{{% resource_link "1c58078e-c216-4f74-b40b-0d18df627824" "A Glycine-dependent Riboswitch that Uses Cooperative Binding to Control Gene Expression" %}}." *Science* 306, no. 5694 (2004): 275-9.
2. Winkler, W. C., et al. "{{% resource_link "e7c523b8-0b49-467f-b3cf-03f6e4ef9049" "Control of Gene Expression by a Natural Metabolite-responsive Ribozyme" %}}." *Nature* 428, no. 6980 (2004): 281-6.
3. Sudarsan, N., et al. "{{% resource_link "b38e14b7-a88c-4289-87b1-69bddbee1861" "An mRNA Structure in Bacteria that Controls Gene Expression by Binding Lysine" %}}." *Genes Dev* 17, no. 21 (2003): 2688-97. \[Open Access\]
4. Winkler, W., et al. "{{% resource_link "b38e14b7-a88c-4289-87b1-69bddbee1861" "Thiamine Derivatives Bind Messenger RNAs Directly to Regulate Bacterial Gene Expression" %}}." *Nature* 419, no. 6910 (2002): 952-6.